### DISCUSSION TITLE
quiz 4.1.4 question hints

### DISCUSSION THREAD

**Discussion by bonosa** [Lightning-AI/dl-fundamentals#19] [Lecture videos and quizzes] [answered]
Hi Sebastian,

Could you please clarify?

thanks,
saroj

**Comment by rasbt** [comment, marked as answer]
<img width="606" alt="Screenshot 2023-01-25 at 2 47 01 PM" src="https://user-images.githubusercontent.com/5618407/214686724-1869de5e-27f1-42f0-8772-542ad44ff9d4.png">
you mean this one? Ohh... it looks like the formatting broke. Will ping our web dev! Sorry!

**Reply by bonosa** [reply]
Gosh! I thought my brain was broken 😂😂😂thanks for clarifying
